domain/relation/service/relation.go

```diff
@@ -152,15 +169,20 @@ func (s *Service) GetRelationUnit(
 	return "", errors.NotImplemented
 }
 
-// GetRelationApplicationSettings returns the application settings for the
-// given unit and relation identifier combination.
+// GetRelationUnitSettings returns the unit settings for the relation
+// identifier.
 func (s *Service) GetRelationUnitSettings(
 	ctx context.Context,
 	relationUnitUUID corerelation.UnitUUID,
 ) (map[string]string, error) {
 	return nil, errors.NotImplemented
 }
 
+// GetRelationUUIID returns the relation UUID for the given relation ID.
+func (s *Service) GetRelationUUIID(ctx context.Context, relationID int) (corerelation.UUID, error) {
+	return "", errors.NotImplemented
+}
+
```

Comment:
todo: delete me. Relation IDs and relation UUIDs are both unique identifiers for the relation. The service methods should be setup to use the correct identifier for the task and where it's used. 

Given that we won't use implementing the Suspended functionality.